packages/aws-lambda/lambdas/runtime_emulator/build-and-run.sh

```diff
@@ -73,6 +73,7 @@ docker run \
   --env INSTANA_TIMEOUT=$instana_timeout \
   --env INSTANA_ENDPOINT_URL=$instana_endpoint_url \
   --env INSTANA_AGENT_KEY=$instana_agent_key \
+  #--env LAMBDA_HANDLER='esm/app.handler' \
```

Comment:
_nitpick:_ Do we need this line for testing esm? Do you want to add a comment here as well to explain that?

Reply:
Yes for esm. I will leave another comment :)